src/Api/AdminConsole/Controllers/OrganizationAuthRequestsController.cs

```diff
@@ -73,7 +76,26 @@ public async Task BulkDenyRequests(Guid orgId, [FromBody] BulkDenyAdminAuthReque
         }
     }
 
-    private async Task ValidateAdminRequest(Guid orgId)
+    [RequireFeature(FeatureFlagKeys.BulkDeviceApproval)]
+    [HttpPost("")]
+    public async Task UpdateManyAuthRequests(Guid orgId, [FromBody] IEnumerable<OrganizationAuthRequestUpdateManyRequestModel> model)
+    {
+        await ValidateAdminRequest(orgId);
+
+        await _updateOrganizationAuthRequestCommand.UpdateManyAsync(
+            orgId,
+            model.Select(x =>
+                new OrganizationAuthRequestUpdateCommandModel
+                {
+                    Id = x.Id,
+                    Key = x.Key,
+                    Approved = x.Approved
+                }
```

Comment:
I like putting this in the request class itself: `model.Select(x => x.ToOrganizationAuthRequestUpdate`.

Reply:
Implemented as suggested on https://github.com/bitwarden/server/pull/4077/commits/977186351714537cf6eca6db1d724be48faf31c8